app.js

```diff
@@ -1,6 +1,6 @@
 const obj = {
   name:"Gowthami",
   email:"ngowthami86@gmail.com",
-  place:"chittor"
+  place:"chittor-517424"
```

Comment:
Remove pin from here and place it as separate field

Reply:
pin removed from place field and added as separate field